src/example-relay/src/graphql/locations/types/output/AddLocationOrError.js

```diff
@@ -0,0 +1,28 @@
+// @flow strict-local
+
+import { GraphQLUnionType } from 'graphql';
+
+import AddLocationError from './AddLocationError';
+import AddLocationMutationResponse from './AddLocationMutationResponse';
+import type { LocationInput } from '../../locations';
+
+export class ValidLocationResponse {
+  location: $ReadOnly<{| ...LocationInput, +id: string |}>;
+  constructor(location: LocationInput) {
+    this.location = {
+      ...location,
+      id: location.locationId,
+    };
+  }
+}
+
+export default new GraphQLUnionType({
+  name: 'AddLocationOrError',
```

Comment:
```suggestion
  name: 'AddLocationResult',
```

Reply:
I was trying to follow the naming convention suggested from [this artsy](https://artsy.github.io/blog/2018/10/19/where-art-thou-my-error/) article.

Reply:
OK, why not. In PD we came up with "Result" postfix so I suggested it here.